cfspopcon/formulas/separatrix_operational_space/LH_transition_power.py

```diff
@@ -0,0 +1,61 @@
+"""Routines to calculate the separatrix power required to reach the LH transition."""
```

Comment:
Needs parameters in docstring https://github.com/cfs-energy/cfspopcon/pull/12#discussion_r1465390902